keras/layers/convolutional.py

```diff
@@ -2258,22 +2258,24 @@ def call(self, inputs):
 class DepthwiseConv2D(Conv2D):
   """Depthwise 2D convolution.
 
-  Depthwise convolution is a type of convolution in which a single convolutional
-  filter is apply to each input channel (i.e. in a depthwise way).
-  You can understand depthwise convolution as being
-  the first step in a depthwise separable convolution.
-
-  It is implemented via the following steps:
+  Depthwise convolution is a type of convolution in which each input channel is
+  convolved with a different filter of depth 1 (called a depthwise kernel). You
```

Comment:
This is inaccurate: you can configure the depth, it doesn't have to be depth 1

Reply:
Can you elaborate on this? I don't see an option for this in the DepthwiseConv2D class.

Reply:
That's the `depth_multiplier` argument.

Reply:
I see what you mean and added a commit for this.